swagger.yaml

```
@@ -46,19 +391,239 @@ paths:
           description: "Services warming up or degraded (at least one check in WARNING or CRITICAL status)"
         500:
           $ref: "#/responses/InternalError"
```

I think this is an issue with the code generation. Both the `429` and the `500` responses should return the `Health` JSON response:

```suggestion
          description: Services is warming up or degraded (at least one check in WARNING or CRITICAL status)
          schema:
            $ref: "#/definitions/Health"
        500:
          description: Service is failing having at least one check with a CRITICAL status having exceeded the recovery grace period.
          schema:
            $ref: "#/definitions/Health"
```

---

This has now been amended